CONTRIBUTING.md

```diff
@@ -7,14 +7,17 @@ Thanks for your interest in contributing to Turbo!
 - [Contributing to Turbo](#contributing-to-turbo)
   - [Contributing to Turborepo](#contributing-to-turborepo)
     - [Building Turborepo](#building-turborepo)
-    - [Testing Turborepo](#testing-turborepo)
+    - [Running Turborepo Tests](#running-turborepo-tests)
+      - [Rust Tests](#rust-tests)
   - [Debugging Turborepo](#debugging-turborepo)
   - [Benchmarking Turborepo](#benchmarking-turborepo)
   - [Updating `turbo`](#updating-turbo)
   - [Publishing `turbo` to the npm registry](#publishing-turbo-to-the-npm-registry)
   - [Contributing to Turbopack](#contributing-to-turbopack)
+    - [Turbopack Architecture](#turbopack-architecture)
     - [Testing Turbopack](#testing-turbopack)
     - [Benchmarking Turbopack](#benchmarking-turbopack)
+    - [Profiling Turbopack](#profiling-turbopack)
```

Comment:
TOC was updated when I saved the file. I don't know what plugin I have that is doing this magic.